sanity(docs): updating the docs (#414)
litmuschaos/litmus-docs: docs/chaosengine-concepts.md

</tr>
</table>

<table>
<tr>
  <th>Field</th>
  <td><code>.spec.components.runner.nodeSelector</code></td>
</tr>
<tr>
  <th>Description</th>
  <td>Node selectors for the runner pod</td>
</tr>
<tr>
  <th>Type</th>
  <td>Optional</td>
</tr>
<tr>
  <th>Range</th>
<td>Labels in the from of label key=value</td>
</tr>
<tr>
  <th>Default</th>
  <td><i>n/a</i></td>
</tr>
<tr>
  <th>Notes</th>
  <td>The <code>.spec.components.runner.nodeSelector</code> The nodeselector contains labels of the node on which runner pod should be scheduled. Typically used in case of infra/node level chaos.</td>
</tr>
</table>

<table>
<tr>
  <th>Field</th>
  <td><code>.spec.components.runner.tolerations</code></td>
</tr>
<tr>
  <th>Description</th>
  <td>Toleration for the runner pod</td>
</tr>
<tr>
  <th>Type</th>
  <td>Optional</td>
</tr>
<tr>
  <th>Range</th>
<td><i>user-defined</i> (type: []corev1.Toleration)</td>
</tr>
<tr>
  <th>Default</th>
  <td><i>n/a</i></td>
</tr>
<tr>
  <th>Notes</th>

>   <td>The <code>.spec.components.runner.tolerations</code> The tolerations contains toleration for the runner pod so that it will scheduled on the given node. Typically used in case of infra/node level chaos.</td>

Reviewer: ```suggestion
  <td>The <code>.spec.components.runner.tolerations</code> Provides tolerations for the runner pod so that it can be scheduled on the respective tainted node. Typically used in case of infra/node level chaos.</td>
```

</tr>
<tr>
  <th>Notes</th>
  <td>The <code>.spec.components.experimentannotation</code> allows developers to specify the custom annotations for the experiment pod.</td>
</tr>
</table>

<table>
<tr>
  <th>Field</th>
  <td><code>.spec.experiments[].spec.components.tolerations</code></td>
</tr>
<tr>
  <th>Description</th>
  <td>Toleration for the experiment pod</td>
</tr>
<tr>
  <th>Type</th>
  <td>Optional</td>
</tr>
<tr>
  <th>Range</th>
<td><i>user-defined</i> (type: []corev1.Toleration)</td>
</tr>
<tr>
  <th>Default</th>
  <td><i>n/a</i></td>
</tr>
<tr>
  <th>Notes</th>

>   <td>The <code>.spec.components.tolerations</code> The tolerations contains toleration for the experiment pod so that it will scheduled on the given node. Typically used in case of infra/node level chaos.</td>

Reviewer: ```suggestion
  <td>The <code>.spec.components.tolerations</code>Tolerations for the experiment pod so that it can be scheduled on the respective tainted node. Typically used in case of infra/node level chaos.</td>
```
